resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -288,6 +288,16 @@ The following is an example to rerun tests with the pre-built image.
         -Dspark.kubernetes.test.imageTag=2022-03-06 \
         'kubernetes-integration-tests/runIts'
 
+You can also specify your specific dockerfile to build JVM/Python/R based image to test:
+
+    build/sbt -Psparkr -Pkubernetes -Pkubernetes-integration-tests \
+        -Dtest.exclude.tags=minikube \
+        -Dspark.kubernetes.test.deployMode=docker-desktop \
+        -Dspark.kubernetes.test.DockerFile=/path/to/Dockerfile \
+        -Dspark.kubernetes.test.pyDockerFile=/path/to/Dockerfile.py \
+        -Dspark.kubernetes.test.rDockerFile=/path/to/Dockerfile.r \
+        'kubernetes-integration-tests/test'
```

Comment:
Could you move this to the end of this doc, @Yikun ?

Reply:
done, :)

Reply:
Although this is a minor comment, it would be great to be consistent with the above examples.
- First, this is not setting `deployMode=docker-desktop`. If then, why do we have `-Dtest.exclude.tags=minikube`?
- Second, in the same way, we had better use `-Dspark.kubernetes.test.imageTag=2022-03-06` consistently.

Reply:
> First, this is not setting deployMode=docker-desktop. If then, why do we have -Dtest.exclude.tags=minikube?

we set the docker-desktop, maybe some display error on github?

> Second, in the same way, we had better use -Dspark.kubernetes.test.imageTag=2022-03-06 consistently.

Sure